htdocs/accountancy/journal/bankjournal.php

```diff
@@ -785,17 +786,17 @@
 	include DOL_DOCUMENT_ROOT . '/accountancy/tpl/export_journal.tpl.php';
 
 	// CSV header line
-	print '"' . $langs->trans("BankId").'"' . $sep;
-	print '"' . $langs->trans("Date") . '"' . $sep;
-	print '"' . $langs->trans("PaymentMode") . '"' . $sep;
-	print '"' . $langs->trans("AccountAccounting") . '"' . $sep;
-	print '"' . $langs->trans("LedgerAccount") . '"' . $sep;
-	print '"' . $langs->trans("SubledgerAccount") . '"' . $sep;
-	print '"' . $langs->trans("Label"). '"' . $sep;
-	print '"' . $langs->trans("Debit") . '"' . $sep;
-	print '"' . $langs->trans("Credit") . '"' . $sep;
-	print '"' . $langs->trans("Journal")  . '"' . $sep;
-	print '"' . $langs->trans("Note")  . '"' . $sep;
+	print '"' . html_entity_decode($langs->trans("BankId")).'"' . $sep;
```

Comment:
If you want to have the correct value of a tranlsation instead of the html entities coding value, you have just to do this:
$langs->transnoentitiesnoconv("Date")
instead of
html_entity_decode($langs->trans("PaymentMode"))